static-site/src/components/Footer/sitemap.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import styled from "styled-components";
+import Link from 'next/link'
+import { FaExternalLinkAlt } from "react-icons/fa";
+import * as Styles from "../splash/styles";
+import { BigSpacer } from "../../layouts/generalComponents";
+
+
+const UnstyledList = styled.ul`
+  list-style: none;
+`
+
+const ListItem = styled(Styles.FocusParagraph)`
+  line-height: 0.7;
+`
+
+const UncoloredLink = styled.span`
+  & a {
+    color: #000 !important;
+  }
+`
+
+const FooterIcon = styled.span`
+  margin-left: 2px;
+  font-size: 12px;
+`
+
+export const SiteMap = () => {
+  return  (
+    <>
+      <div className="col-md-3">
```

Comment:
I think these (bootstrap) parameters aren't quite right for the sitemap as it switches to a single column too soon:

<img width="973" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/0af5ce56-b13a-4380-a106-7c1f8486bee0">